extensions/amp-video-iframe/0.1/amp-video-iframe.js

```diff
@@ -126,6 +127,12 @@ class AmpVideoIframe extends AMP.BaseElement {
      * @private
      */
     this.boundOnMessage_ = (e) => this.onMessage_(e);
+
+    /**
+     * @param {!Element} element
+     * @return {!Promise<IntersectionObserverEntry>} element
+     */
+    this.measureIntersection = (element) => measureIntersection(element);
```

Comment:
Sorry, but what's the value of this instance property?

Reply:
Just for stubbing. Is it possible to stub a module using sinon?

Reply:
I see. This is a bit annoying. Maybe we can just stub the whole `IntersectionObserver` for this? If not, then I think we should at least make it a package-private or private member.

Reply:
I'll stub in this case, but as a general pattern it feels too leaky (and laborious) for each client of `measureIntersection` to do that? Ideally they could just stub the function (the way Services would have allowed)